nuxeo-java-client-test/src/test/resources/test-portal-sso-login-contrib.xml

```diff
@@ -0,0 +1,62 @@
+<component name="org.nuxeo.ecm.platform.authenticator.portal.sso.config">
+
+    <require>org.nuxeo.ecm.platform.ui.web.auth.WebEngineConfig</require>
+    <require>org.nuxeo.ecm.platform.login.Portal</require>
+    <require>org.nuxeo.ecm.automation.server.auth.config</require>
+
+    <require>org.nuxeo.ecm.restapi.server.auth.config</require> <!-- specificChains -->
+
+    <extension target="org.nuxeo.ecm.platform.ui.web.auth.service.PluggableAuthenticationService" point="authenticators">
```

Comment:
No need to override the default `PORTAL_AUTH`plugin contributed in the `org.nuxeo.ecm.platform.login.portal` bundle deployed in the test.